### ISSUE TITLE
[BUG] Pip vs. Conda install of darts["torch"]

### ISSUE THREAD

**Issue by martinb-bb** [unit8co/darts#983]
Installing darts through Pip vs. conda causes different behaviour: see PR https://github.com/OpenBB-finance/OpenBBTerminal/pull/1885#issuecomment-1141122775

```python
numeric_forecast = predicted_values.quantile_df()["AdjClose_0.5"].tail(n_predict)
print(numeric_forecast)
```

**Through Pip (dataarray format issue): pip install "u8darts[torch]"**

![image](https://user-images.githubusercontent.com/105685594/170995588-73ae2921-fb47-424a-bf0f-0c8d44a77615.png)

**Through conda (No issue): conda install -c conda-forge -c pytorch u8darts-torch**
<img width="417" alt="image" src="https://user-images.githubusercontent.com/105685594/171016300-158a618c-a2dc-425c-bca8-17f361e5ab89.png">

Pip is modifying some specific library that is causing issues with how we process the DF.
It is making it an xarray object vs. float64

In summary: 
`quantile_df()` behaves differently, giving your a different types of output depending on the way you install Darts.

**Comment by martinb-bb** [comment]
Update: I have found the culprit to the problem with Error: unsupported format string passed to DataArray.__format__

It has to do with how pandas manipulates<class 'pandas.core.series.Series'>. This has been updated in later versions of pandas.

Seemed to be an issue with how Poetry was managing dependancies.  It would install pandas (1.3.5)  not latest  (1.4.2).

Closing issues now. 👍
